tests/bdd/steps/_compiled_feature_steps.py

```diff
@@ -0,0 +1,91 @@
+"""
+This code is auto-generated.
+
+From /home/jon/projects/lms/tests/bdd/steps/feature_steps/.
```

Comment:
Do you want to enshrine your local path here :) ?

Reply:
Hmm, nope. Looks like there are two things going on here. The first is absolute paths, which isn't great, the other part might be unstable sort ordering.

There's a third you haven't got to, which is it's hard to write black compliant code automatically. I've tried, but I can't quite nail it. The result is that you often have un-committed changes, but not if you run `make format`.

I'll see what I can do about the order and relative path for now.

Reply:
I've fixed the local names and sorted alphabetically, this should sort most of the issues hopefully.